internal/api/v1/admin/contacts/admin_contact_filter_query_builder.go

```diff
@@ -0,0 +1,28 @@
+package contacts
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/querybuilders"
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/contacts"
+	"github.com/bitcoin-sv/spv-wallet/models/filter"
+)
+
+type adminContactFilterBuilder struct {
+	contactFilter filter.AdminContactFilter
+}
+
+func (p *adminContactFilterBuilder) Build() (url.Values, error) {
+	builder := contacts.ContactFilterQueryBuilder{
+		ContactFilter:      p.contactFilter.ContactFilter,
+		ModelFilterBuilder: querybuilders.ModelFilterBuilder{ModelFilter: p.contactFilter.ModelFilter},
+	}
+	params, err := builder.BuildExtendedURLValues()
+	if err != nil {
+		return nil, fmt.Errorf("failed to build extended URL values: %w", err)
+	}
+
+	params.AddPair("xpubId", p.contactFilter.XPubID)
```

Comment:
Please take a closer look at the JSON tag.
```
field XPubID *string `json:"xpubid,omitempty" example:"623bc25ce1c0fc510dea72b5ee27b2e70384c099f1f3dce9e73dd987198c3486"`
```


There’s a naming discrepancy that should be addressed on the SPV Wallet API models side. For now, we should stick to using **xpubid**.

Reply:
models bumped to beta.39 with fix for that tag